.github/workflows/tasklist-ci-test-reusable.yml

```diff
@@ -41,7 +41,6 @@ jobs:
   integration-tests:
     name: Test
     runs-on: gcp-perf-core-16-default
-    if: ${{ !startsWith(inputs.branch, 'fe-') && !startsWith(inputs.branch, 'renovate/') }}
```

Comment:
@houssain-barouni just to double check - it was intended now run these tests on the Renovate PRs as well? (no problems with that, just checking)

Reply:
@maxdanilov yes, this was brought from the legacy tasklist repository where we had Renovate only configured for Frontend-updates (`dependabot` was used for maven). This check is now obsolete